runWPT.sh

```diff
@@ -1,27 +1,47 @@
-#!/bin/sh
+#!/bin/bash
+# shellcheck disable=SC2155
 
-LOG_DIR="logs"
-LOG_FILE="$LOG_DIR/$(date +%s).log"
-readonly LOG_DIR LOG_FILE
+# usage: HEADLESS=true ./runWPT.sh
+
+readonly HEADLESS="${HEADLESS:-false}"
+readonly LOG_DIR="${LOG_DIR:-logs}"
+readonly LOG_FILE="$LOG_DIR/$(date +%s).log"
+readonly MANIFEST="${MANIFEST:-MANIFEST.json}"
+readonly PRODUCT="${PRODUCT:-chromium}"
+readonly WPT_REPORT="${WPT_REPORT:-wptreport.json}"
+
+if [[ "$HEADLESS" == "true" ]]; then
+  WPT_METADATA="wpt-metadata/mapper/headless"
+else
+  WPT_METADATA="wpt-metadata/mapper/headful"
+fi
+
+echo "--> Running WPT..."
 
 # Go to the project root folder.
 (cd "$(dirname "$0")/" && \
 # Create a folder to store logs.
 mkdir -p "$LOG_DIR" && \
-npm run build && \
 ./wpt/wpt run \
   --binary "$WPT_BROWSER_PATH" \
   --webdriver-binary runBiDiServer.sh \
   --webdriver-arg="--verbose" \
   --webdriver-arg="--log-path=$LOG_FILE" \
-  --webdriver-arg="--headless=true" \
-  --log-wptreport wptreport.json \
-  --manifest wpt/MANIFEST.json \
-  --metadata wpt-metadata/mapper/headless \
-  --timeout-multiplier 8 \
-  chromium \
+  --webdriver-arg="--headless=$HEADLESS" \
+  --log-wptreport "$WPT_REPORT" \
+  --manifest "$MANIFEST" \
+  --metadata "$WPT_METADATA" \
+  "$PRODUCT" \
   "$@")
 
+echo "--> Updating expectations..."
```

Comment:
If you want, we could also add an `UPDATE_EXPECTATIONS` variable to control whether we run this part.

Reply:
Done